core/templates/pages/topic-editor-page/modal-templates/questions-list-select-skill-and-difficulty-modal.component.ts

```diff
@@ -0,0 +1,129 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for questions list select skill and
+ * difficulty modal.
+ */
+
+import { Component, Input, OnInit } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { AppConstants } from 'app.constants';
+import { ConfirmOrCancelModal } from 'components/common-layout-directives/common-elements/confirm-or-cancel-modal.component';
+import { QuestionsListConstants } from 'components/question-directives/questions-list/questions-list.constants';
+import { SkillDifficulty } from 'domain/skill/skill-difficulty.model';
+import { Skill, SkillBackendDict } from 'domain/skill/SkillObjectFactory';
+
+interface Summary {
+  'id': string;
+  'description': string;
```

Comment:
Single quotes with property names are not required.

Reply:
Done.